app/views/components/inputMonthAndYear.scala.html

```diff
@@ -0,0 +1,80 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(govukDateInput: GovukDateInput,
+        span: components.span)
+
+@(
+        form: Form[_],
+        legendContent: String,
+        legendClasses: String,
+        id: String,
+        hintText: Option[String],
+        legendAsPageHeading: Boolean,
+        legendHiddenContent: Option[String])(implicit messages: Messages)
+
+@monthError = @{
+    if(form(s"$id.month").value.contains("")){ "govuk-input--error"} else {""}
+}
+
+@yearError = @{
+    if(form(s"$id.year").value.contains("")){ "govuk-input--error"} else {""}
+}
+
+@allError = @{
+    if(monthError && !yearError && form(s"$id").hasErrors) {"govuk-input--error"} else {""}
+}
+
+    @govukDateInput(DateInput(
```

Comment:
Could you please align this as this should be at the start of line 41, and rest of the related code as well?